### ISSUE TITLE
Issue with BTSettl library

### ISSUE THREAD

**Issue by aaelli99** [mfouesneau/pystellibs#13]
I've recently installed the pystellibs package and am trying to use the BTSettl libraries. I tried to import, like the BaSel and Kurucz libraries and I get an error that says that the "bt-settl.medres.grid.fits" does not exist. I checked the source code and there is no medres file either. Not sure what to do here.

**Comment by mfouesneau** [comment]
Yes, I need to fix this. The files are large, so I put them outside the package, here
https://keeper.mpdl.mpg.de/d/9be3be80d8bc4abe9deb/
